CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
     * Fixed bug where multiline commands were having leading and ending spaces stripped. This would mess up quoted
     strings that crossed multiple lines.
     * Fixed a bug when appending to the clipboard where contents were in reverse order
+    * Fixed issue where run_pyscript failed if the script's filename had 2 or more consecutive spaces
+    * Fixed issue where completer function of disabled command would still run
```

Comment:
Good fix - that would be frustrating if you can tab complete something that you can’t run because it is disabled